spi_nand_flash/src/nand.c

```diff
@@ -320,6 +339,16 @@ esp_err_t spi_nand_flash_write_sector(spi_nand_flash_device_t *handle, const uin
     if (dhara_map_write(&handle->dhara_map, sector_id, buffer, &err)) {
         ret = ESP_ERR_FLASH_BASE + err;
     }
+#if CONFIG_NAND_FLASH_VERIFY_WRITE
+    if (ret == ESP_OK) {
+        ret = s_verify_write(handle, sector_id, buffer, &err);
+        if (ret == ESP_OK) {
+            ESP_LOGI(TAG, "%s: Write buffer verification succeeded", __func__);
```

Comment:
Logging something on success is probably unnecessary, this will just flood the output. Let's log only on failed verification.